### PR TITLE
Add the guide for launch_screen element to config-editor.md

### PR DESCRIPTION
**Pull request by juku1999**
### Change Description ###
Add the guide for launch_screen element to config-editor.md

### CODE REVIEW

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 1]
```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
...
+		<tr>
+			<td>Used to indicate the splash screen.
+			<p><strong>Occurrences:</strong></p>
+			<ul>
+				<li>0 or 1</li>
+			</ul>
+			<p><strong>Expected children:</strong></p>
+			<ul>
+				<li><code>tizen:ls_default</code>
+				<p>Used to indicate the default of the splash screen.</p>
```
Comment (surbhig12) [inline docs/application/tizen-studio/web-tools/config-editor.md]: Suggestion: Can we add value?
Used to indicate the default value of the splash screen.

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 2]
```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
...
+					<li><code>image_border</code>: mandatory, string</li>
+				</ul>
+				</li>
+			</ul>
+			<p><strong>Attributes:</strong></p>
+			<ul>
+				<li><code>ready_when</code>
+				<p>Mandatory; string (available values: <code>first-paint</code> (default), <code>complete</code>, <code>custom</code>)</p>
+				<ul>
+					<li><code>first-paint</code>: hide splash screen when "frame,rendred" callback is called</li>
```
Comment (surbhig12) [inline docs/application/tizen-studio/web-tools/config-editor.md]: Suggest: Please add "the" and use code tag:

hide splash screen when the `frame,rendred` callback is called

Also, please verify the spelling of rendered in the name of the callback.

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 3]
```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
...
+					<li><code>complete</code>: hide splash screen when "load,complete" callback is called</li>
```
Comment (surbhig12) [inline docs/application/tizen-studio/web-tools/config-editor.md]: Suggest: Please add "the" and use code tag:

hide splash screen when the `load,complete` callback is called

File: docs/application/tizen-studio/web-tools/config-editor.md [hunk 4]
```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
...
+					<li><code>custom</code>: hide splash screen when window.screen.show() method was called</li>
```
Comment (surbhig12) [inline docs/application/tizen-studio/web-tools/config-editor.md]: Suggest rephrasing to:

hide splash screen when the `window.screen.show()` method is called

Comment (surbhig12) [reply docs/application/tizen-studio/web-tools/config-editor.md]: @juku1999,
Please verify the comments. Also the content for "Tizen Launch Screen" occurs twice in the same file. Please verify.

**Comment by surbhig12** [comment]
@juku1999,

Please verify the comments. Thank you.

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 1 above)
Comment (juku1999) [reply docs/application/tizen-studio/web-tools/config-editor.md]: > Used to indicate the default value of the splash screen.
done

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 2 above)
Comment (juku1999) [reply docs/application/tizen-studio/web-tools/config-editor.md]: done

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 3 above)
Comment (juku1999) [reply docs/application/tizen-studio/web-tools/config-editor.md]: done

File: docs/application/tizen-studio/web-tools/config-editor.md (hunk 4 above)
Comment (juku1999) [reply docs/application/tizen-studio/web-tools/config-editor.md]: Suggest rephrasing to:
hide splash screen when the window.screen.show() method is called
=> done

Please verify the comments. Also the content for "Tizen Launch Screen" occurs twice in the same file. Please verify.
=> The contents are for mobile and wearable profile.

**Pull request merged**